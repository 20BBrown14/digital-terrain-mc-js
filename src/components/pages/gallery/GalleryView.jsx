import React from 'react';

/**
 * Gallery page view.
 */
const GalleryView = () => (
  <div>
    Gallery Page
  </div>
);

export default GalleryView;
