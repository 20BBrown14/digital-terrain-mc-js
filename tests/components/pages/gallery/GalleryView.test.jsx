import React from 'react';
import GalleryView from '../../../../src/components/pages/gallery/GalleryView';

describe('GalleryView', () => {
  it('renders a default view', () => {
    const view = render(<GalleryView />);
    expect(view).toMatchSnapshot();
  });
});
