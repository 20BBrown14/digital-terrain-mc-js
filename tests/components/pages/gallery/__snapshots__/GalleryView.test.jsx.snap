// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GalleryView renders a default view 1`] = `
<div>
  Gallery Page
</div>
`;
